src/components/login/google-login/google-login.jsx

```diff
@@ -1,6 +1,11 @@
 ﻿import React from "react";
 import { GoogleLogin as Login } from "react-google-login";
 import PropTypes from "prop-types";
+import "../google-facebook-login.scss";
```

Comment:
Move this import to the bottom of other imports.
All CSS style imports need to placed after all modules imports.

Reply:
Okay, thank you, moved.